db/migrate/20211123060535_create_treatment_inertia_tables.rb

```diff
@@ -0,0 +1,35 @@
+class CreateTreatmentInertiaTables < ActiveRecord::Migration[5.2]
+  def up
+    create_table :clean_medicine_to_dosage, id: false do |t|
+      t.bigint :rxcui, null: false
+      t.string :medicine, null: false
+      t.float :dosage, null: false
+    end
+    execute "COPY clean_medicine_to_dosage FROM '#{Rails.root}/config/data/treatment-inertia/clean_medicine_to_dosage.csv' WITH DELIMITER ',' CSV HEADER"
+
+    create_table :raw_to_clean_medicine, id: false do |t|
```

Comment:
This name needs pluralisation because rails. Others also.

Reply:
As of now, all these lookup tables will only be used in the matview creation, and I don't see any need to use this from Rails. However, I'll consider this in the next pass while renaming tables. Might be a small thing to address.

Reply:
Pluralised with https://github.com/simpledotorg/simple-server/pull/3085.